examples/conf-quickstart/druid/middleManager/runtime.properties

```diff
@@ -11,10 +11,6 @@ druid.indexer.task.baseTaskDir=var/druid/task
 # HTTP server threads
 druid.server.http.numThreads=9
 
-# Processing threads and buffers
-druid.processing.buffer.sizeBytes=256000000
-druid.processing.numThreads=2
```

Comment:
Isn't it configured as `druid.indexer.fork.property.druid.processing.numThreads=2`